### PR TITLE
Make the lane context dump on crashes more pretty

### PR DESCRIPTION
**Pull request by KrauseFx**
screenshot 2016-12-02 13 45 35

### CODE REVIEW

File: fastlane/lib/fastlane/lane_manager.rb [hunk 1]
```diff
@@ -180,5 +179,25 @@ def self.load_dot_env(env)
...
+      # Print a nice table unless in $verbose mode
+      rows = Actions.lane_context.collect do |key, content|
+        [key, content.to_s]
+      end
+      rows = FastlaneCore::PrintTable.limit_row_size(rows)
+
+      require 'terminal-table'
+      puts Terminal::Table.new({
+        title: "Lane Context".yellow,
+        rows: rows,
```
Comment (milch) [inline fastlane/lib/fastlane/lane_manager.rb]: Appease the ruby style gods

Comment (KrauseFx) [inline fastlane/lib/fastlane/lane_manager.rb]: Looks beautiful to me :P

Comment (milch) [inline fastlane/lib/fastlane/lane_manager.rb]: `rubocop` begs to differ:

```
lib/fastlane/lane_manager.rb:199:19: C: Avoid comma after the last item of a hash.
        rows: rows,
```

**Comment by KrauseFx** [comment]
@milch this is fixed now

**Review by mfurtak — APPROVED** [review]

**Pull request merged**